src/config.ts

```diff
@@ -2,8 +2,7 @@
 
 /// <reference path="./typings.d.ts" />
 
-import Brolog from 'brolog'
-import readPkgUp from 'read-pkg-up'
+import { log } from 'brolog'
```

Comment:
What's the reason for removing the separate logger of wechaty-puppet-padpro?

Reply:
For ... saving memory ...

Indeed, it's an unrelated minor change, if we need a separate logger for padpro, please feel free to just revert this change.

Reply:
Okay, let's keep this "memory saving" change.